packages/common/src/build/context.js

```diff
@@ -0,0 +1,7 @@
+export default class BuildContext {
+  constructor(builder) {
+    this.nuxt = builder.nuxt
```

Comment:
Idea stolen from @clarkdo :P We can do `Object.assign(this, DEFAULTS, builder)`

Reply:
I thought about this, it may assign all properties in builder, I feel it's too much 😄